Fixed #32819 -- Established relationship between form fields and their help text. (#16920)
django/django: docs/releases/5.0.txt

* The new ``assume_scheme`` argument for :class:`~django.forms.URLField` allows
  specifying a default URL scheme.

> * To improve accessibility and enable screen readers to associate form fields
>   with their help text, the form field now includes a new HTML attribute
>   ``aria-describedby``, which points to the corresponding help text HTML
>   element.

Reviewer: `aria-describedby` is not a new HTML attribute. Also, the last sentence sounds redundant:
```suggestion
* In order to improve accessibility and enable screen readers to associate form
  fields with their help text, the form field now includes ``aria-describedby``
  HTML attribute.
```

* The new ``assume_scheme`` argument for :class:`~django.forms.URLField` allows
  specifying a default URL scheme.

* To improve accessibility :attr:`~django.forms.Field.help_text` is now
  associated to its input using ``aria-describedby``.

* To improve accessibility and enable screen readers to associate form fields
  with their help text, the form field now includes a new HTML attribute

>   ``aria-describedby``, which points to the corresponding help text.

Reviewer 2: ```suggestion
  ``aria-describedby``, which points to the corresponding help text HTML element.
```

Author: Fixed.

* The new ``assume_scheme`` argument for :class:`~django.forms.URLField` allows
  specifying a default URL scheme.

* To improve accessibility :attr:`~django.forms.Field.help_text` is now

>   associated to its input using ``aria-describedby``.

Reviewer 3: Note the association is the other way around, `aria-describedby` is on the element being described (the form control), not the help text.

>   associated to its input using ``aria-describedby``.

Author: Is 

```To improve accessibility and enable screen readers to associate form fields
  with their help text, the form field now includes a new HTML attribute
  ``aria-describedby``, which points to the corresponding help text.```

fine?

>   associated to its input using ``aria-describedby``.

Reviewer 2: Looks good, yes, but do note that the diff seems to have two bullet points now for the same thing? unless I'm missing something!

>   associated to its input using ``aria-describedby``.

Author: I have removed the duplicated text.
